www/index.html

```diff
@@ -1,73 +1,72 @@
 <!DOCTYPE html>
 <html>
 
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
-    <title></title>
+    <head>
```

Comment:
Please don't bundle code style refactors like whitespace such as this into feature pull requests